README.md

```diff
@@ -283,6 +283,28 @@ Port 2: Hub used for Raspberry Pi (45b:209 / 1-10.2)
 
 ```
 
+## Waiting For Devices
+
+If you have a device that might take a moment or two to come up (block devices for example) the script can be told to 
+hold off outputting all of the name=value pairs until the requested name is present. This can be done using the `--wait-for-env`/`-w` flag. This flag accepts a list of values e.g. `-w FIRST_NAME SECOND_NAME` etc
+
+Using our raspberry-pi example above if we wanted to wait for its UART to come up we would type:
+``` Bash
+$ lsudt -x -i pci-0000:00:14.0-usb-0:5 -w RASPBERRY_PI_UART_0 
+```
+  
+And this would stop the anything from being printed to stdout before `RASPBERRY_PI_UART_0` is generated. We can also ask for multiple envs to be present and it will wait until all are present before printing:
+``` Bash
+$ lsudt -x -i pci-0000:00:14.0-usb-0:5 -w RASPBERRY_PI_UART_0 RASPBERRY_PI_POWER_0
+```
+
+By default the script will try 10 times (once a second). If you wish to CHANGE this then:
+``` Bash
+$ lsudt -x -i pci-0000:00:14.0-usb-0:5 -w RASPBERRY_PI_UART_0 -o 0 # Only try once
```

Comment:
That's not how this works, setting -o to 0 results in an intimidate return. Please drop this line. Users can use -o 1 to try only once, or -o 0 to now at all.